roles/telemetry_verify_metrics/tasks/check_compute_node_containers.yml

```diff
@@ -0,0 +1,7 @@
+- delegate_to: "{{ compute_node }}"
```

Comment:
I'm still not a fan of having this in the role, but I'm not going to block based on this.

I hope you'll later consider whether this is better being included in the ci/run_verify_metrics_osp18.yml playbook, as a separate play for compute and controller node tests.
Something similar (i.e. separating compute and controller tests into different plays) is done in the logging jobs.